packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -16,6 +16,7 @@
 
 import { inject, injectable } from 'inversify';
 import URI from '@theia/core/lib/common/uri';
+import * as path from 'path';
```

Comment:
it is not allowed to use node.js apis in browser cone, please use core URI and Path